C4SGWeb/src/app/user/account/user-account.component.ts

```diff
@@ -162,6 +164,26 @@ export class UserAccountComponent implements OnInit {
 
   }
 
+
+  openModal(user) {
+      this.modalActions.emit({action: "modal", params: ['open']});
+      this.selectedUser=user;
+
+    }
+
+    closeModal() {
+      this.modalActions.emit({action: "modal", params: ['close']});
+    }
+
+    deleteUser(user:User): void{
+      console.log(user.id);
+      this.userService.delete(user.id).subscribe(
+        error => console.log(error)
+      );
+
```

Comment:
Thanks for the comment. 
1) I will fix the subscribe function and resubmit the PR. 
2) Do you want me to use windows confirm() instead of materialize modals?